art/art_param.py

```diff
@@ -27,8 +27,10 @@
 OVERWRITE_TYPE_ERROR = "The 'overwrite' type must be bool."
 SEP_TYPE_ERROR = "The 'sep' type must be str."
 SPACE_TYPE_ERROR = "The 'space' type must be int."
+DETAILED_RETURN_TYPE_ERROR = "The '__detailed_return' type must be bool."
 ART_ENVIRONMENT_WARNING = "[Warning] '{0}' is not printable in this environment."
-FONT_ENVIRONMENT_WARNING = "[Warning] '{0}' is not printable in this environment."
```

Comment:
I am not sure about `aprint` if we should update it, too. @sepandhaghighi what do you think?

Reply:
We will handle this one in another PR 💯